<launch>
	<arg name="veh" doc="Name of vehicle. ex: megaman"/>
	<arg name="pkg_name" value="auto_localization"/>
    <arg name="node_name" default="bot_detector_node"/>

    <arg name="bot" default="true" doc="true for finding robot, false for getting background"/>
    <arg name="det_th" default="1000000" doc="setting threshold from Duckiebot detection"/>

    <remap from="image" to="camera_node/image/rect"/>
    <group ns="$(arg veh)">
    	<node name="bot_detector_node" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen" clear_params="true" required="true">
    	<param name="bot" value="$(arg bot)"/>
        <param name="det_th" value="$(arg det_th)"/>
    	</node>
	</group>

</launch>